blackbox/docs/appendices/release-notes/4.0.0.rst

```diff
@@ -0,0 +1,424 @@
+.. _version_4.0.0:
+
+=============
+Version 4.0.0
+=============
+
+Released on 2019/06/25.
+
+.. NOTE::
+
+    If you are upgrading a cluster, you must be running CrateDB 3.0.4 or higher
+    before you upgrade to 4.0.0.
+
+    We recommend that you upgrade to the latest 3.2 release before moving to
```

Comment:
i kept this recommendation

Reply:
why not `3.3`?

Reply:
I was not really sure what was this recommendation about, so kept it as it is, but it seems it is here just to minimize gap between versions? Updating it now.